virt_lightning/api.py

```diff
@@ -442,31 +467,60 @@ def redirect_request(self, req, fp, code, msg, hdrs, newurl):
     logger.debug("Date: %s", last_modified)
     size = r.headers["Content-Length"]
     logger.debug("Size: %s", size)
-    lenght = int(r.headers["Content-Length"])
+    length = int(r.headers["Content-Length"])
```

Comment:
You can use `int(size)` to avoid duplication access to headers.

Reply:
He's actually just fixed my typo :-D.